css/style.css

```diff
@@ -94,3 +95,263 @@ header {
   padding: 20px 10px;
   margin: auto 0;
 }
+
+/* section works tonic1 */
+
+.btn {
+  color: #396df2;
+  box-sizing: border-box;
+  border-radius: 8px;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 18px;
+  line-height: 24px;
+  text-align: center;
+  letter-spacing: 0.05em;
+  padding: 15px;
+  margin: 0 10px;
+}
+
+.btn-enabled-tonic1 {
+  color: #6070ff;  /* #396DF2 */
+  background: #fff;
+  border: 1px solid #7f8cff;
+}
+
+.btn-enabled-tonic1:hover {
+  color: #fff;
+  background: #6070ff; /* * */
+  box-shadow: 0 8px 16px rgba(70, 85, 255, 0.25);
+}
+
+.btn-enabled-tonic1:active {
+  color: #fff;
+  background: #2230d2;
+}
+
+.btn-disable {
+  color: #5e6c84;
+  border: 1px solid #c1c7d0;
+}
+
+.tags-ul-tonic1 {
+  display: flex;
+  flex-flow: row nowrap;
+  gap: 10px;
+}
+
+.tag-li-tonic1 {
+  padding: 6px 14px; /* orasini ochish */
+  background: #ebebff;
+  border-radius: 8px;
+  letter-spacing: 0.05em; /* nimaga */
+  font-style: normal;
+  font-weight: bold;
+  font-size: 12px;
+  line-height: 15px;
+  list-style: none; /* nuqtalani oladi listdan */
+}
+
+.tag-li-tonic1 a {
+  color: #6070ff;
+  text-decoration: none;  /* tagidagi chiziqlarni o'chirish */
+}
+
+.p-tonic1 {
+  color: #344563;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 20px;
+  line-height: 24px;
+  margin: 12px 0;
```

Comment:
- [ ] Please set the `font size `of paragraphs as given on Figma design, it should be `15px`

Your design  | Figma design
------------- | -------------
![image](https://user-images.githubusercontent.com/77038610/143584829-666be437-d14c-4a34-8804-181fe46029bd.png) |![image](https://user-images.githubusercontent.com/77038610/143584878-2ec00bfd-7bb1-4ece-998d-43e59fe75a11.png)